resources/analysisTools/qcPipeline/chrom_diff.r

```diff
@@ -19,8 +20,14 @@ for(i in 1:nrow(data)){
   v[i] <- sum(data[i,])
 }
 
-pdf(opt$outputFile, height = 10, width = 10)
-
-barplot(v, names.arg = rownames(data), col = rainbow(nrow(data)),ylab = "# reads", xlab = "Chromosome", main = plot_title)
+# fix for https://github.com/DKFZ-ODCF/AlignmentAndQCWorkflows/issues/63
+oldWD = getwd()
+setwd(dirname(opt$outputFile))
+tmpFileName=paste0(UUIDgenerate(),".tmp")
 
+pdf(tmpFileName, height = 10, width = 10)
+  barplot(v, names.arg = rownames(data), col = rainbow(nrow(data)),ylab = "# reads", xlab = "Chromosome", main = plot_title)
```

Comment:
Here is s.th. with the indentation.

Reply:
the indentation is intended (to visually state which comments are related to the pdf)
-->
openFile
   doSomethingWithTheFile
closeFile

Reply:
Although I am generally not a friend of pure indentation-based blocks (like in Python and -- to my dismay -- since 3.0 also in Scala), R is generally not capable of it. I am thinking in particular of automatic indentation and IDE support.

Maybe something like

```R
withOpenFile(...) {
   doSomethingWithTheFile
}
```

is possible in R. Didn't use it for quite some time. This would give you the semantics of structuring plus the automatic file handling (but that's not really a benefit here at the end of the script :-P).

Reply:
Otherwise maybe 

```R
{
  open()
  do something
}
```

???

Reply:
I removed the indentation